src/EditorFeatures/TestUtilities/Diagnostics/DiagnosticTaggerWrapper.cs

```diff
@@ -128,15 +128,15 @@ public void Dispose()
             _registrationService.Unregister(_workspace);
         }
 
-        public async Task WaitForTags()
+        public async Task WaitForTags(bool willBlockOnCompletion)
```

Comment:
i can't find anyone who passes false in for this.

Reply:
Parameter removed.